src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/doc_viewer_span_overview/span_overview.tsx

```diff
@@ -60,49 +65,59 @@ export function SpanOverview({
   const transactionId = flattenedDoc[TRANSACTION_ID_FIELD];
 
   return (
-    <TransactionProvider transactionId={transactionId} indexPattern={transactionIndexPattern}>
-      <FieldActionsProvider
-        columns={columns}
-        filter={filter}
-        onAddColumn={onAddColumn}
-        onRemoveColumn={onRemoveColumn}
-      >
-        <EuiPanel color="transparent" hasShadow={false} paddingSize="none">
-          <EuiSpacer size="m" />
-          <SpanSummaryTitle
-            spanName={flattenedDoc[SPAN_NAME_FIELD]}
-            formattedSpanName={formattedDoc[SPAN_NAME_FIELD]}
-            spanId={flattenedDoc[SPAN_ID_FIELD]}
-            formattedSpanId={formattedDoc[SPAN_ID_FIELD]}
-          />
-          <EuiSpacer size="m" />
-          {spanFields.map((fieldId) => (
-            <SpanSummaryField
-              key={fieldId}
-              fieldId={fieldId}
-              fieldConfiguration={fieldConfigurations[fieldId]}
+    <RootTransactionProvider
+      traceId={flattenedDoc[TRACE_ID_FIELD]}
+      indexPattern={tracesIndexPattern}
+    >
+      <TransactionProvider transactionId={transactionId} indexPattern={tracesIndexPattern}>
+        <FieldActionsProvider
+          columns={columns}
+          filter={filter}
+          onAddColumn={onAddColumn}
+          onRemoveColumn={onRemoveColumn}
+        >
+          <EuiPanel color="transparent" hasShadow={false} paddingSize="none">
+            <EuiSpacer size="m" />
+            <SpanSummaryTitle
+              spanName={flattenedDoc[SPAN_NAME_FIELD]}
+              formattedSpanName={formattedDoc[SPAN_NAME_FIELD]}
+              spanId={flattenedDoc[SPAN_ID_FIELD]}
+              formattedSpanId={formattedDoc[SPAN_ID_FIELD]}
+              showActions={showActions}
             />
-          ))}
-
-          {spanDuration && (
-            <>
-              <EuiSpacer size="m" />
-              <SpanDurationSummary
-                spanDuration={spanDuration}
-                spanName={formattedDoc[SPAN_NAME_FIELD]}
-                serviceName={formattedDoc[SERVICE_NAME_FIELD]}
+            <EuiSpacer size="m" />
```

Comment:
Would it make sense to use <EuiFlexGroup> instead of manually adding the <EuiSpacer>?